src/pybind/mgr/dashboard/frontend/src/app/shared/configurations/ngx-bootstrap-tooltip.ts

```diff
@@ -0,0 +1,8 @@
+import { TooltipConfig } from 'ngx-bootstrap';
+
+export function getFormTooltipConfig(): TooltipConfig {
+  return Object.assign(new TooltipConfig(), {
+    placement: 'auto',
+    delay: 500
```

Comment:
As mentioned by @ricardoasmarques we have a component for help popovers. IMHO popovers are easier to use on touch devices than tooltips.